src/routes/scout/endgame/+page.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+	import { derived } from 'svelte/store';
+
+	let driverquickness: number;
+	let driverfieldawareness: number;
+	let broken: boolean;
+	let functional: boolean;
+	let notes: string;
+	function submit() {}
+</script>
+
+<div class="flex flex-col items-start align-top h-dvh p-3 accent-text_yellow">
```

Comment:
1. items start is redundant b/c it's the default
2. align-top is used for inline or table-cell box and has no effect on flexboxes, you're looking for items-start (which is the default on flexboxes)
3. it would be better to use min-h-dvh in case everything doesn't fit on the screen